File: k11-orientacao-a-objetos-em-java/logica/ImprimePadrao5.java
class ImprimePadrao5 {
	public static void main(String[] args) {
		int penultimo = 0;
		int ultimo = 1;

		System.out.println(penultimo);
		System.out.println(ultimo);

		for(int contador = 0; contador < 28; contador++) {
			int proximo = penultimo + ultimo;
			System.out.println(proximo);

			penultimo = ultimo;
			ultimo = proximo;
		}
	}
}

File: k02-desenvolvimento-web-com-html-css-e-javascript/cap-04/javascript/imprime-padrao-5.js
var penultimo = 0;
var ultimo = 1;

document.writeln(penultimo);
document.writeln('<br />');
document.writeln(ultimo);
document.writeln('<br />');

for(var contador = 0; contador < 28; contador++) {
	var proximo = penultimo + ultimo;
	document.writeln(proximo);
	document.writeln('<br />');

	penultimo = ultimo;
	ultimo = proximo;
}